opennmt/utils/hooks.py

```diff
@@ -5,6 +5,7 @@
 import io
 import time
 import six
+import numpy as np
```

Comment:
Looks like this import is no longer used.

Reply:
Sorrryy! My bad.